### PR TITLE
Add software delivery page

### PR DESCRIPTION
**Pull request by mattgraygithub**
This adds an index page for the software delivery service line, as per [leankit ticket 221](https://codurance-online.leankit.com/card/1069431621) and adds all of the copy.

Also did some cleanup of the translations by removing duplicate entries for the 'How we solve your problems' section and the contact CTA's, since the copy for these is the same across all service lines.

Also added the Software Modernisation publications 

Notes/still to do for the software delivery page overall:
- Change tabber nav bar styling so that the nav items stretch across the entire block
- Find out about design changes to text in hero section and make changes if necessary 
- Add icons and images
- Add link to this page from services page
- change metarobots and sitemap info

### CODE REVIEW

**Review by Dan-Bird — APPROVED** [review]
Changes look good to me. 

From looking at the page in the browser there are a few things to tidy up on the template and tabber after this is merged in.

File: src/services/software-delivery/index.html [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+metarobots: noindex,nofollow
+sitemap: false
+
+intro:
+  background-image-small: /assets/custom/img/service-lines/service-line-example/meta-social-image.png
+  background-image-large: /assets/custom/img/service-lines/service-line-example/meta-social-image.png
+  foreground-image-large: /assets/custom/img/service-lines/service-line-example/meta-social-image.png
+
+curated_publications:
+location: 'soft-mod'
```
Comment (LL782) [inline src/services/software-delivery/index.html]: What do you think about changing this location to `software-delivery-page` or something similar?

We would then have to add that location to the posts that we want to show in this location but I think it keeps it more understandable if each page is it's own location.

**Review by LL782 — APPROVED** [review]
Code looks good. I haven't checked in the browser. 

One comment but doesn't need to be addressed before merging necessarily.

File: src/assets/custom/css/_sass/_services-curated-publications.scss [hunk 2]
```diff
@@ -6,7 +6,7 @@
   }
 }
 
-.services__curated-publications--one-publication {
+.services__curated-publications--small-margin-top {
```
Comment (LL782) [inline src/assets/custom/css/_sass/_services-curated-publications.scss]: Not sure about this rename? Want to make a case for it? Not a biggie if you want to merge but I think the classname should describe the what the element is not how it should be styled.

**Review by LL782 — APPROVED** [review]
Just re-reviewed and all the latest commits look good! LGTM

File: src/assets/custom/css/_sass/_services-curated-publications.scss (hunk 2 above)
Comment (mattgraygithub) [reply src/assets/custom/css/_sass/_services-curated-publications.scss:9]: Struggling to think of another name! - it's basically the curated publications if there's only one publication OR if there is no webinar. Any ideas?

**Pull request merged**